modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -473,6 +474,8 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::checkInputWindow(const GenericReal<is_ad>
       mooseWarning(msg.str());
     else if (behavior == WindowFailure::ERROR)
       mooseError(msg.str());
+    else
```

Comment:
...this turned into an else if, with an `else mooseError("internal error...blah blah)`

Reply:
I changed this over to your commit with a switch statement here.